src/modules/currencyWallets/api.js

```diff
@@ -382,6 +382,7 @@ export function makeCurrencyApi (
 
```

Comment:
line 380.. should this be `lt(sub(max, min)`.... Not `lte`. If min and max are still different by one then you need to try both options to see which fails otherwise you'll have an off by one error.